views/apartments/show.erb

```diff
@@ -1 +1,16 @@
-This is info about apartment #3.
+<ul>
+<li>
+  <h2> <%= @apartment.address %></h2>
```

Comment:
I'm not sure it's syntactically correct to put an `h2` inside of an `li`... if you want to change the style, I'd use a span with a class here.

Even better, don't include the address in the `ul`... if it's important, put it as an `h2` above the list.